ansible/roles/operator-pipeline/tasks/main.yml

```diff
@@ -1,34 +1,4 @@
 ---
-- name: Creates and init service accounts
```

Comment:
No need to create the service account for Tekton:
```
The OpenShift Pipelines Operator adds and configures a Serviceaccount named pipeline that has sufficient permissions to build and push an image. This ServiceAccount is used by pipelineruns.
```
https://openshift.github.io/pipelines-docs/docs/0.10.5/assembly_creating-applications-with-cicd-pipelines.html